### DISCUSSION TITLE
Slow in browsing NAS

### DISCUSSION THREAD

**Discussion by clriis** [rern/rAudio#1649] [Fixed Issues]
With the latest (20240331) update, my RPi 3B has become slow in initiating NAS browsing.

**Comment by R101G** [comment]
I have the same problem with USB folders. There is a delay of up to 10 seconds after I click on the library USB icon.

I have downloaded the new image, and it's the same.

**Reply by rern** [reply]
@R101G,

Please try the UPDATE.

**Comment by clriis** [comment]
Thanks. That solved the issue.

**Comment by R101G** [comment]
OK for me too, thanks. 
(I did get that error warning following the update, by the way.)

**Reply by rern** [reply]
@R101G,

Please post the output of `/srv/http/bash/status.sh` with the track which displayed the error.

**Reply by R101G** [reply]
@rern 
This refers to #1645. If you are asking what track was in the playlist while updating, I do not know, but it was probably an RP stream. The error warning does not always occur, but I will post the information next time it does.
